app/workers/vic/submission_job.rb

```diff
@@ -11,6 +11,7 @@ def perform(vic_submission_id, form, user_uuid)
 
       @vic_submission_id = vic_submission_id
       parsed_form = JSON.parse(form)
+      Raven.tags_context(parsed_form: parsed_form)
```

Comment:
Are there any potentially sensitive fields in this `parsed_form` that we want to be careful about inserting into Sentry context?

Reply:
the sensitive fields are being filtered
http://sentry.vetsgov-internal/vets-gov/platform-api-development/issues/33439/